Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -632,6 +632,7 @@ private record TargetPaths(string Directory, string Filename) {
       Contract.Requires(dafnyProgram != null);
       Contract.Assert(dafnyProgramName != null);
 
+      // TODO: The outputWriter parameter is unused.  Remove it?
```

Comment:
Really? I see `outputWriter.WriteLine("Program compiled successfully")` below for e.g., just guarded by `--compile-verbose`

Reply:
Really: specifically, it's never used by *callers*; it always get its default value `null`, which then routes to Console.Out

Reply:
Oh I see. If you end up having to push any more commits, it would be great to clarify that in the comment. But I wouldn't spend another CI run on it. :)

Reply:
```suggestion
      // TODO: outputWriter seems to always be passed in as `null`.  Remove it?
```